### DISCUSSION TITLE
Sales Contribution from extra Features

### DISCUSSION THREAD

**Discussion by DongHarry-Kang** [google/lightweight_mmm#271] [Q&A]
Hi I am new beginner of Lightweight. I realized that in the lightweight outcome, the sales contribution of extra features is not shown anywhere. Is there anyway I can plot all channels including extra features in one graph? for the below analysis, I have 2 channels + 2 extra features (owned channels with no cost); but in the result, the contribution of extra features are not shown

![image](https://github.com/google/lightweight_mmm/assets/146807827/06533e43-e12a-4526-b466-c92d705efd81)

**Comment by statm3n** [comment]
Confirm you are right - the **effect** of extra features are reflected in baseline demand/contribution. You could hack the fitted mmm object to get the **effect** of extra features. Here, I would be cautious using the term 'contribution' for extra features and be cautious what extra features to be included in MMM.

**Reply by DongHarry-Kang** [reply]
Hi Thank you very much. Could you share me the code on how to get contribution on extra futures?
